app/api/search/search.js

```diff
@@ -56,6 +56,37 @@ function agregationProperties(properties) {
   });
 }
 
+function snippetsFromSearchHit(hit) {
+  let snippets = [];
+  if (hit.inner_hits && hit.inner_hits.fullText.hits.hits.length) {
+    const regex = /\[\[(\d+)\]\]/g;
+
+    const fullTextHighlights = hit.inner_hits.fullText.hits.hits[0].highlight;
+
+    snippets = fullTextHighlights[Object.keys(fullTextHighlights)[0]].map((snippet) => {
```

Comment:
I suggest extracting this `Object.keys(fullTextHighlights)[0]` to a variable:
`const fullTextLanguageKey = Object.keys(fullTextHighlights)[0]`

Reply:
Alright